README.md

```diff
@@ -223,6 +224,22 @@ Note, this will only specifically enable the `RequestLogger`.
 - [Updating diagrams](/docs/updating-diagrams.md)
 - [Setting up a CircleCI context for deployment](/docs/setting-up-circleci-context-for-deployment.md)
 
+## External dependencies
```

Comment:
```suggestion
### External dependencies
```
Should this be living under the documentation heading?

Reply:
Done